src/app/services/constants.ts

```diff
@@ -893,3 +893,14 @@ export const CLOZE_ITEM_SECTION_ID = "non-selected-items";
 export const CLOZE_DROP_ZONE_ID_PREFIX = "drop-zone-";
 // Matches: [drop-zone], [drop-zone|w-50], [drop-zone|h-50] or [drop-zone|w-50h-200]
 export const dropZoneRegex = /\[drop-zone(?<params>\|(?<index>i-\d+?)?(?<width>w-\d+?)?(?<height>h-\d+?)?)?]/g;
+
+export const USER_ROLES: Role[] = [
+  "STUDENT",
+  "TUTOR",
+  "TEACHER",
+  "EVENT_LEADER",
+  "CONTENT_EDITOR",
+  "EVENT_MANAGER",
+  "ADMIN",
+];
+export type UserRole = (typeof USER_ROLES)[number];
```

Comment:
Maybe I'm getting confused by the TypeScript syntax, but this one does not look like a constant, right? This is a new type. Shouldn't this live somewhere else? For instance, we already have a `services/user.ts`, and there is another type called `UserRoleAndLoggedInStatus` in that file, so perhaps this one can be moved to the top of said file. But you know the codebase better than me, perhaps there is a more appropriate place where this can be moved.

Reply:
Yeah that makes more sense, relocating